### PR TITLE
Print JS and PY implementation

### PR DESCRIPTION
**Pull request by gaudyb**
- Print verb javascript implementation
- Print verb python implementation
- Workflow and jest tests

### CODE REVIEW

**Review by natoverse — CHANGES_REQUESTED** [review]
Can you add one more optional arg to the verb, `limit`, with a default of 10? 10 would match arquero. I believe pandas `toString` may be the entire table, so by default for logging we will want a safe limit (calling `print(df)` defaults to 5 rows).

In arquero you can pass the `limit` arg as `limit` in the `table.print()` function, in pandas this is `max_rows` in the `df.toString()` function.

File: javascript/react/src/components/StepDescription/selectStepDescription.ts [hunk 1]
```diff
@@ -61,6 +61,7 @@ const descriptions = {
 	join: JoinDescription,
 	lookup: LookupDescription,
 	merge: MergeDescription,
+	print: NoParametersDescription,
```
Comment (AlonsoGuevara) [inline javascript/react/src/components/StepDescription/selectStepDescription.ts:64]: Should this be PrintDescription?

File: javascript/workflow/src/verbs/print.ts [hunk 2]
```diff
@@ -0,0 +1,21 @@
+/*!
+ * Copyright (c) Microsoft. All rights reserved.
+ * Licensed under the MIT license. See LICENSE file in the project.
+ */
+import type { PrintArgs } from '@datashaper/schema'
+
+import type { ColumnTableStep } from './util/factories.js'
+import { stepVerbFactory } from './util/factories.js'
+
+export const DEFAULT_LIMIT_ROWS = 10
```
Comment (natoverse) [inline javascript/workflow/src/verbs/print.ts:10]: You can put this in workflow/src/verbs/defaults so all default verb args are centralized

**Review by natoverse — CHANGES_REQUESTED** [review]

**Review by natoverse — COMMENTED** [review]
Did you commit the print.ts defaults file? I see the index export but not the file it references

**Comment by natoverse** [comment]
> Did you commit the print.ts defaults file? I see the index export but not the file it references

Oh nevermind, maybe it was a previous commit

**Review by natoverse — APPROVED** [review]

**Pull request merged**